content/en/docs/tasks/administer-cluster/migrating-from-dockershim/change-runtime-containerd.md.md

```diff
@@ -0,0 +1,133 @@
+---
+title: "Migrating Container runtime from dockershim to containerd"
+weight: 10
+content_type: task 
+---
+
+- Drain Node
```

Comment:
Do readers need to do this just once, or repeat this for each node? We should make that detail clear.